packages/core/src/logger.ts

```diff
@@ -41,6 +42,15 @@ class Logger implements LoggerInterface {
     return isBrowser ? console.error(str) : process.stderr.write(colorize.red(str));
   }
 
+  errorWithStack(str: string | Error) {
```

Comment:
Do we really need this method? Why not simply output the stacktrace where we need it?